examples/umbrella/sn2.xyz

```diff
@@ -0,0 +1,440 @@
+6
```

Comment:
This is a lot of lines for a single molecule..

Reply:
oops